### PR TITLE
add git command

### PR DESCRIPTION
**Pull request by Sispheor**

### CODE REVIEW

File: eg/examples/git.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    git init
+    git add README.md
+    git commit -m "first commit"
+    git remote add origin https://github.com/me/myrepo.git
+    git push -u origin master
+
+
+
+# Branches
+Create a new branch and switch to it
```
Comment (scorphus) [inline eg/examples/git.md]: Hello, @Sispheor! You forgot to add one blank line just after this heading

File: eg/examples/git.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    git push origin :<branchname>
+
+Delete branch from remote repository
+
+    git push origin --delete <branchname>
+
+
+
+# Tags
+Create a tag on the last commit
```
Comment (scorphus) [inline eg/examples/git.md]: And here too.

File: eg/examples/git.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    git push --tags
+
+Create a tag on a on a specific commit
+
+    git tag 1.0.0 <commitID>
+
+
+
+# Undo
+Go back to the last commit
```
Comment (scorphus) [inline eg/examples/git.md]: And here too.

**Comment by scorphus** [comment]
@Sispheor any updates on this, sir?

**Comment by srsudar** [comment]
Good timing! I'm plann

**Comment by srsudar** [comment]
My bad. Clumsy fingers. 

I'm planning on merging this this week, likely with some heavy editing. Git is obviously a very tricky command to get right.

**Comment by srsudar** [comment]
Let me know if you guys think what I ended up is missing something or is a significant lessening of @Sispheor 's original commit. I had to change quite a bit to keep in-line with the style of the other commits. I also looked through the StackOverflow `git` questions that are upvoted with scores over 1000 and added examples for questions that seemed like relevant rather than specialized behavior.

**Comment by scorphus** [comment]
@srsudar: What an impressive addition! You improved on top of @Sispheor's already amazing job. It covers so many topics and in the right level of detail. Thank you both!

But, while reading it, I noticed an inconsistent pattern not only in this but across many of the examples. Please check my last pull request.

**Pull request merged**